tst/test-error/func-and-proc-call-trace.g

```diff
@@ -0,0 +1,96 @@
+informproc0 := function(l)
+Add(l,1);
+l();
+return; end;
```

Comment:
Actually, why the `return;`? It is redundant, isn't it?

Perhaps also add a second semicolon after `end;`, to suppress printing the function, to reduce clutter in the `.out` file.